targets/gen_noro_nowcasts_targets.R

```diff
@@ -50,21 +51,62 @@ gen_noro_nowcasts_targets <- list(
   ),
   # Forecast quantiles as a scoringutils forecast object
   tar_target(
-    name = quantiled_nowcast_noro,
+    name = su_quantile_noro,
     command = scoringutils::as_forecast_quantile(
-      data = su_forecast_sample,
-      probs = config$Hub_quantiles
+      data = su_sample_noro,
+      probs = config$norovirus$quantiles
     )
   ),
   # Get a wide dataframe with only 50th and 95th for plotting
   tar_target(
     name = summary_nowcast_noro,
-    command = quantiled_nowcast_noro |>
+    command = su_quantile_noro |>
       filter(quantile_level %in% c(0.025, 0.25, 0.5, 0.75, 0.975)) |>
```

Comment:
why have these in the config if going to hard code?

Reply:
These are the ones i want to plot, the ones in the config are the ones we want to use for scoring (though noting that I had assumed this would be a subset... but since noro only scores on 50th and 90th it won't be...) I can make a separate entry in config for quantiles for plotting?